schemas/mobility.schema

```diff
@@ -0,0 +1,25 @@
+SET client_min_messages TO WARNING;
+BEGIN;
+
+DROP SCHEMA IF EXISTS $PG_SCHEMA CASCADE;
+CREATE SCHEMA $PG_SCHEMA_MOBILITY;
+
+CREATE TABLE $PG_SCHEMA_MOBILITY.mobility (
+locale_id  					   INTEGER NOT NULL,
+timestamp  					   TIMESTAMP NOT NULL,
```

Comment:
It will probably be my downfall one day, but I tend to favor names that are as short as possible. What do you think about `ts` here? An added benefit is you don't need to escape the column name in queries.

Reply:
I think that makes a lot of sense, will update accordingly.